### ISSUE TITLE
Accept HEIC (HEIF) format as picture upload

### ISSUE THREAD

**Issue by StanoRiga** [RobinLinus/snapdrop#304]
Hi all.

Referring to: https://support.apple.com/en-us/HT207022
> When you share media using other methods, such as AirDrop, Messages or email, it may be shared in a more compatible format, such as JPEG or H.264, depending on whether the receiving device supports the newer media format.

(Hope I get this right.)
Apple does convert *.heic images into *jpg images if the webform does not support *.heic as picture. Unfortually this comes along with loss of quality and meta-data.
My feature request is to change the website so that a mobile browser on an iPhone supports *.heic as picture type, and apple does not convert it anymore into *.jpg.

Workaround: Store the image in *.heic format somewhere online or on your phone, then select "file" instead of "photo/video" when you start the transfer. Select the *.heic file. Then the picture is transferred in its original version.

Thank you very much.

**Comment by fm-sys** [comment]
I think the current behavior is intentional because otherwise there would be no compatibility with Windows/Android OS

**Comment by Bellisario** [comment]
Yes, in fact this is an auto conversion made by Safari, so, we can do anything.

**Comment by StanoRiga** [comment]
The conversion is done by chrome and other browsers as well. And yes, it is intention of iOS to ensure compatibility if the target can not handle this format. But as far as I understand you can tell iOS within the Website to accept their format, so the conversation will not be performed. And in cases where you transfer to windows (for expanple) heic is meanwhile supported. 
Actually there is no way to transfer photos from iOS to anywhere without loss of quality (besides my described workaround). 
I might be wrong, but if I understand the infos that I received via websearch, this can be avoided with a extended web implementation regarding the file types.

**Comment by bleomycin** [comment]
Support for this would be greatly appreciated. Having the image automatically degraded when it's not 100% necessary is unfortunate.
